src/VideoCommand.cc

```diff
@@ -207,6 +214,12 @@ Json::Value AddVideo::construct_responses(
     return ret;
 }
 
+bool AddVideo::need_blob(const Json::Value& cmd)
+{
+    const Json::Value& add_video_cmd = cmd[_cmd_name];
+    return !(add_video_cmd.isMember("from_local_file"));
```

Comment:
from_local_file should be a define. But I am sure we are not using defined for other similar cases. We can probably do a future pass and make all the cases defines. for now it should be good.

Reply:
I agree with that. I guess a single patch that defines all the parameters would do it.